src/commands/generate/serviceGenerator.ts

```diff
@@ -222,3 +229,13 @@ function mediaType(conjureType?: IType) {
     }
     return "MediaType.APPLICATION_JSON";
 }
+
+function parsePathParamsFromPath(httpPath: string): string[] {
```

Comment:
perhaps we could use a regex like https://github.com/palantir/conjure-typescript-runtime/blob/develop/packages/conjure-client/src/fetchBridge/fetchBridge.ts#L166 to extract the path params?

Reply:
so I could replace this method with 
```
[... httpPath.matchAll(/\{([^:\}]+)([:]*[^\}]*)\}/g)].map(x => x[1])
```

but not convinced that's easier to read